css/style.css

```diff
@@ -1,189 +1,233 @@
-:root{
-    --grey-dark: #363434;
-    --grey: #adadad;
-    --white: #fff;
-    --orange: #f05a28;
-
+:root {
+  --yellow: #ffcc5c;
+  --beige: #ffeead;
+  --grey: #adadad;
+  --white: #fff;
+  --orange: #f05a28;
 }
 body {
-    font-family: 'Roboto', sans-serif;
-    -webkit-font-smoothing: antialiased;
-    margin: 0;
-    padding: 0;
-    border: 0;
-    box-sizing: border-box;
-    font-weight: 400;
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+  margin: 0;
+  padding: 0;
+  border: 0;
+  box-sizing: border-box;
+  font-weight: 400;
 }
 header {
-    display: flex;
-    justify-content: space-between;
-    align-items: center;
-    padding: 1rem 3rem;
-    border-bottom: 0.1rem solid hsl(0, 0%, 75%);
+  display: flex;
+  margin: 0;
+  padding: 0;
+  justify-content: space-between;
+  align-items: center;
+  padding: 1rem 3rem;
+  border-bottom: 0.1rem solid hsl(0, 0%, 75%);
 }
 main {
-    display: flex;
-    flex-direction: column;
-    margin-top: -60px;
- }
-
- .logo{
-    display: block;
-    min-width: 1.5rem;
-    min-height: 1.5
-    rem;
- }
+  display: flex;
+  flex-direction: column;
+  margin-top: -80px;
```

Comment:
The layout will be simpler to develop if you remove this line, and also remove the [margin-top on the `section` selector](https://github.com/CodeYourFuture/HTML-CSS-Module-Project/pull/517/commits/ddbbba3580693f159c7b26eb2d110ae20542af4d#diff-1fc556f95754ee7e33d91044125c44bb9f750c99be4406756ffb27413adfcaf5R109).
Remember, it's best not to style using element selectors, as it will affect every one of them and then you might have to add more CSS to work against those styles, as you have here.